hw/ip/hmac/data/hmac.hjson

```diff
@@ -3,6 +3,7 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name:               "hmac",
+  one_line_desc:      "SHA-2 based keyed hash message authentication code engine, unhardened",
```

Comment:
```suggestion
  one_line_desc:      "Accelerator for SHA-256 backed HMAC and the SHA-256 hash function",
```

I think HMAC only supports SHA-256 so perhaps we make that explicit. It is also not clear from the description that you can also use it to just generate SHA-256 hashes, it might be worth adding that.

@vogelpi do we need to mention unhardened in one line the summary? Would it be cleaner to mention countermeasures only when they are present?

Reply:
I tried to not go too much into the details and instead wanted to make sure to spell out the HMAC acronym. Now I realized that I also didn't do this for AES. I think your suggestion sounds good to me @mundaym .

About the "unhardened" : I thought to be explicit here but maybe it's better to only mention that in the 1-paragraph summary where we have also space to say that if hardening is required, people should refer to a SW implementation for SHA-2 or KMAC. So this sounds good to me as well.

Reply:
Thanks both! After discussing with @vogelpi, we went for
> Accelerator for SHA-256-based keyed hash message authentication code and the SHA-256 hash function